apps/bare-expo/ios/Podfile.lock

```diff
@@ -1338,7 +1338,7 @@ SPEC CHECKSUMS:
   EXWebBrowser: 37388663fa51cbfc883de2244931a5182d948e5f
   FacebookSDK: 4b9bb8e2824898b47f18c666dc145c09b40609e6
   FBLazyVector: c71c5917ec0ad2de41d5d06a5855f6d5eda06971
-  FBReactNativeSpec: 289039c27ca26d02a6683c56c6fc7868bc804c43
+  FBReactNativeSpec: 8758eb0003171e03e3bb263e52848cf4286fe30d
```

Comment:
oh no, this cherry-pick does matter: https://github.com/expo/react-native/commit/f21d982c822eef1f60002857698c9ef7fa920df3

i need to find a way to deal with the unstable hash